websites/I/InfinityBotList/dist/metadata.json

```diff
@@ -0,0 +1,47 @@
+{
+  "$schema": "https://schemas.premid.app/metadata/1.6",
+  "service": "InfinityBotList",
+  "author": {
+    "name": "Toxic Dev",
+    "id": "510065483693817867"
+  },
+  "contributors": [
+    {
+      "name": "Anaxes",
+      "id": "567885938160697377"
+    }
+  ],
+  "description": {
+    "en": "The future of Discord Bot Listing Services!",

```

Comment:
Why is this not the same description matching the metadata?

Reply:
![image](https://user-images.githubusercontent.com/64903135/148419782-ce840ef9-ec6a-4d8f-a225-98f1bf1bea71.png)